ObjectMemory_MarkingCollector.c

```diff
@@ -29,7 +29,7 @@ void ObjectMemory_zeroReferenceCounts(void) {
 
 void ObjectMemory_markAccessibleObjects(void) {
   ObjectPointer rootObjectPointer;
-  while(NO) { /* haven't defined any root objects yet */
+  while(NO) { /* haven't defined any root objects yet, warning expected */
```

Comment:
Could also change the expression to `while(rootObjectPointers)` and get no warning - `rootObjectPointers` is `NULL` so this will have the same effect.